package.json

```diff
@@ -59,12 +59,12 @@
     "eslint-config-standard": "^16.0.0"
   },
   "peerDependencies": {
-    "eslint": ">=7.12.1",
-    "eslint-plugin-import": ">=2.22.1",
-    "eslint-plugin-node": ">=11.1.0",
-    "eslint-plugin-promise": ">=4.2.1",
-    "typescript": ">=3.9",
-    "@typescript-eslint/eslint-plugin": ">=4.0.1"
+    "eslint": "^7.12.1",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
```

Comment:
We should support the same `peerDependencies` as [eslint-config-standard](https://github.com/standard/eslint-config-standard) does.

Here we're not supporting`"eslint-plugin-promise": "^5.0.0"` but [eslint-config-standard](https://github.com/standard/eslint-config-standard) does `"eslint-plugin-promise": "^4.2.1 || ^5.0.0"`.

Reply:
So how come the test is passing?

Reply:
The current dependency, `eslint-config-standard@16` doesn't yet have that wider `eslint-plugin-promise` peer dep. So that issue will become relevant in the PR that upgrades the `eslint-plugin-promise` dependency.

Reply:
The test is actually failing on my local machine.
![image](https://user-images.githubusercontent.com/25207499/122636200-34195e80-d0e8-11eb-8501-4730852cb30e.png)

Also, it seems like, the CI with Travis doesn't seem to work or is really slow to start...
Are you sure, you don't want to migrate to GitHub Actions?

Reply:
#598

Reply:
I agree with @Divlo that it should be `"eslint-plugin-promise": "^4.2.1 || ^5.0.0"` here

```suggestion
    "eslint-plugin-promise": "^4.2.1 || ^5.0.0",
```

Else it would be a regression in what versions are supported + `"eslint-config-standard": "^16.0.0"` has been widened to indicate support for it.